app/components/satis/menu/component.html.slim

```diff
@@ -1,4 +1,4 @@
-div.sts-menu data-controller="satis-menu" data-action="#{menu.event}->satis-menu#show mouseleave->satis-menu#hide"
+div.sts-menu data-controller="satis-menu" data-action="#{menu.event}->satis-menu#show"
```

Comment:
I would keep mouseleave as an option, but add a way to configure close-by-click.

The whole clickOutside thing is quite (over)complicated, what you do is:
- add an action click@document->satis-menu#clickOutside
- in the clickOutside  find the check if the clicked element doesn't have the sts-menu as an ancestor (use `closest`). if it does, the click was inside, if it doesn't it's outside